config/routes.rb

```diff
@@ -410,6 +410,7 @@
       end
 
       namespace :material, path: 'materials' do
+        get '/', to: 'materials#load_default'
```

Comment:
Could change the path to `get 'load_default', to: 'materials#load_default'` for better readability + tracing.

Also default paths for rails are typically routed to functions like `show`, `index` etc. so it might cause confusion

Reply:
Change the path to `/courses/1/materials/load_default` might not the @cysjonathan suggested. Should we keep it ?

Reply:
Yes that's fine